samples/bluetooth/peripheral_hids_mouse/src/main.c

```diff
@@ -161,12 +161,15 @@ static void advertising_continue(void)
 
 	if (!k_msgq_get(&bonds_queue, &addr, K_NO_WAIT)) {
 		char addr_buf[BT_ADDR_LE_STR_LEN];
-		struct bt_conn *conn;
+		struct bt_conn *conn = NULL;
 
-		adv_param = *BT_LE_ADV_CONN_DIR;
+		adv_param = *BT_LE_ADV_CONN_DIR(addr);
 		adv_param.options |= BT_LE_ADV_OPT_DIR_ADDR_RPA;
 		conn = bt_conn_create_slave_le(&addr, &adv_param);
```

Comment:
Need to remove `bt_conn_create_slave_le` as well as the `conn` variable.

Reply:
removed